src/gleam_community/colour.gleam

```diff
@@ -239,12 +239,12 @@ fn rgba_to_hsla(
 
   let max_colour = float.max(r, float.max(g, b))
 
-  let h1 = case True {
-    _ if max_colour == r -> float.divide(g -. b, max_colour -. min_colour)
-    _ if max_colour == g ->
+  let h1 = case max_colour == r, max_colour == g {
```

Comment:
I kind of wish we had elixir's cond but with if multiple clause and else syntax.
the old case way mimicked that.